frontend/static/js/App.js

```diff
@@ -18,13 +18,21 @@ const imageUploadOptions = {
     progressText: "![Загружаю файл...]()",
     urlText: "![]({filename})",
     errorText: "Ошибка при загрузке файла :(",
+    allowedTypes: [
+        // Images
+        "image/jpeg",
+        "image/png",
+        "image/jpg",
+        "image/gif",
```

Comment:
We don't really want gifs to be honest. But let's keep them for now. Maybe in the future I'll convert them to videos 🤔